Tasks/PowerShellV2/task.json

```
@@ -97,6 +97,15 @@
             "helpMarkDown": "If this is true, this task will fail if any errors are written to the error pipeline, or if any data is written to the Standard Error stream. Otherwise the task will rely on the exit code to determine failure.",
             "groupName": "advanced"
         },
+        {
+            "name": "showWarnings",
+            "type": "boolean",
+            "label": "Show warnings as azure devops warnings",
+            "required": false,
+            "defaultValue": "false",
+            "helpMarkDown": "If this is true, and your script writes a warning you pipeline is success but with warnings.",
```

Nit: looks like some typo here, and some semantical also - 'true' value of this flag doesn't guarantee that pipeline will be succeded.
```suggestion
            "helpMarkDown": "If this is true, and your script writes a warnings - they are shown as warnings also in pipeline",
```

---

Uff you are right. 🤦‍♂️ #midnightcode